documentation/technical_writing_slides.md

```diff
@@ -0,0 +1,452 @@
+---
+type: slide
+slideOptions:
+  transition: slide
+  width: 1400
+  height: 900
+  margin: 0.1
+---
+
+<style>
+  .reveal strong {
+    font-weight: bold;
+    color: orange;
+  }
+  .reveal p {
+    text-align: left;
+  }
+  .reveal section h1 {
+    color: orange;
+  }
+  .reveal section h2 {
+    color: orange;
+  }
+  .reveal code {
+    font-family: 'Ubuntu Mono';
+    color: orange;
+  }
+</style>
+
+
+# Technical Writing
+
+---
+
+## 1. Introduction
+
+---
+
+## A Poem
+
+> If people don’t know why your project exists,
+they won’t use it.
+> If people can’t figure out how to install your code,
+they won’t use it.
+> If people can’t figure out how to use your code,
+they won’t use it.
+
+Taken from [Write The Docs](https://www.writethedocs.org/guide/writing/beginners-guide-to-docs/#why-write-docs)
```

Comment:
```suggestion
Taken from [Write the Docs](https://www.writethedocs.org/guide/writing/beginners-guide-to-docs/#why-write-docs)
```

Reply:
They use "Write the Docs".

Reply:
I know that is also what I suggest. You have a capital "T" in there, i.e., "The". The other places capitalization was normally as on the homepage.